content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -34,18 +34,21 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
 
 For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
 
-### Multiple LNS connections for Sigfox and Actility
+### Device Management
 
-The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+### Extensible device registration
 
-![All devices](/images/release-notes/multiple_lns_connection.png)
+Extensible device registration is a new feature that allows microservice developers to implement their own device registration process without explicitly extending a single line of UI source code.
 
-For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
+A custom input mask can be created using JSON schema and some specific endpoints that the microservice needs to provide. This is then finally displayed in a dropdown on the device registration page as soon as the microservice is subscribed to the tenant.
```

Comment:
input mask -> input definition?

Reply:
I am here reffering to the UI form as input mask.. input definition would fit for JSON schema. Not sure...

Reply:
yeah I meant for JSON schema. Input mask sound too technical.

Reply:
We usually call it "editor" in the user docs :-). If this is what you mean here.

Reply:
This is > what does this refer to? Currently it refers to the mask but this is not meant here.
on the device registration page > on the **Device registration** page

Reply:
Not sure if "editor" fits here. What I am trying to refer to is the form generated with the help of JSON schema, because this is what the users see in the end.

This is > should also refer to the form which is displayed to the user, when he clicks the button in the dropdown

Reply:
Maybe we call it simply: _A custom device registration form can be created... .This form is then finally displayed..._